src/reducers/enrollmentReducer.js

```diff
@@ -33,15 +33,17 @@ const handleEnrollment = (state, payload, requestType) => {
     const newState = JSON.parse(JSON.stringify(state));
     switch(requestType) {
         case "GET":{
-            data.forEach(({student, course, id, payment_list, enrollment_balance}) => {
+            data.forEach(({student, course, id, payment_list, enrollment_balance, sessions_left, last_paid_session_datetime }) => {
                 let newStudentData = newState[student] || {};
-                let newCourseData = newStudentData[course] || {
+                let newCourseData = {
                     "enrollment_id": id,
                     "course_id": course,
                     "student_id": student,
                     "notes": {},
```

Comment:
If there were already notes for this enrollment in the redux state, set notes to be those notes (don't wipe them out on enrollment FETCH)